research/src/pages/select/select.proposal.mdx

```diff
@@ -116,11 +116,55 @@ children:
+| `keydown(space)` | Toggles aria-expanded              | [aria-expanded](#https://www.w3.org/TR/wai-aria-1.1/#aria-expanded) attr |
+| `keydown(enter)` | Toggles the open state on the host | [open](#open-state) state                                                |
+| `keydown(enter)` | Toggles aria-expanded              | [aria-expanded](#https://www.w3.org/TR/wai-aria-1.1/#aria-expanded) attr |
+
+### part listbox
+
+| Event                 | Behavior                                                                                                               | Impacts                                                                  |
+| --------------------- | ---------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------ |
+| `keydown(down key)`\* | Moves focus to the next `<option>` in the listbox                                                                      | focus                                                                    |
+| `keydown(up key)`\*   | Moves focus to the previous `<option>` in the listbox                                                                  | focus                                                                    |
+| `keydown(enter)`      | Changes the `selected` state of the current `<option>` and updates the `<select>` value property                       | [selected](#selected) prop <br/> [value](#value) prop                    |
+| `keydown(space)`      | Changes the `selected` state of the current `<option>` and updates the `<select>` value property                       | [selected](#selected) prop <br/> [value](#value) prop                    |
+| `keydown(enter)`      | If the `<select>` does **not** have the `multiple` attribute then toggle the state of `open` on the `<select>`         | [open](#open-state) state                                                |
+| `keydown(enter)`      | If the `<select>` does **not** have the `multiple` attribute then toggle the aria-expanded attribute on the `<select>` | [aria-expanded](#https://www.w3.org/TR/wai-aria-1.1/#aria-expanded) attr |
+| `keydown(space)`      | If the `<select>` does **not** have the `multiple` attribute then toggle the state of `open` on the `<select>`         | [open](#open-state) state                                                |
+| `keydown(space)`      | If the `<select>` does **not** have the `multiple` attribute then toggle the aria-expanded attribute on the `<select>` | [aria-expanded](#https://www.w3.org/TR/wai-aria-1.1/#aria-expanded) attr |
+| `keydown(escape)`     | Toggles the `open` state of the `<select>`                                                                             | [open](#open-state) state                                                |
```

Comment:
There should be a row for `keydown(escape)` to toggle `aria-expanded` too, right?

A refactoring that we could consider (not necessarily part of this change) could be another table that ties the value of certain states to the value of certain `aria` semantics.  E.g. declaring that the value of the `open` state is always reflected in the value of `aria-expanded` would allow us to delete a lot of these rows.